### ISSUE TITLE
Add new chart `times stats` to the `Activity` page

### ISSUE THREAD

**Issue by mgledi** [change-metrics/monocle#784]
Why?
A highly valueable metric for understanding the cycle time of PRs is the "time to merge". There is already the possibility to get the mean "time to merge" for a defined time range. To optimize engineering teams it is important to also see the trend of this and related metrics. 

What?
Add a new chart to the Activity page:
- y-axis is duration
- x-axis is time (days)
- chart contains three lines: "time to merge", "time comment mean time", and "first review mean time"

**Comment by TristanCacqueray** [comment]
In https://github.com/change-metrics/monocle/issues/764 we drafted an improve activity page to include trend (e.g. simple comparison with previous range)

**Comment by mgledi** [comment]
Yes. This is what I did for myself. Have two browser windows side by side. But I was interested in more time points.

**Comment by TristanCacqueray** [comment]
Perhaps such metrics could also be collected through an external tool, e.g. by running the relevant queries using a sliding window or something. Then the web interface could expose such data through a `create report` option?
